src/win/win-helpers.cpp

```diff
@@ -604,8 +604,6 @@ namespace librealuvc
                 if (device_info == INVALID_HANDLE_VALUE)
                     return false;
 
-                auto di = std::shared_ptr<void>(device_info, SetupDiDestroyDeviceInfoList);
```

Comment:
Fix exception when opening device (using `get_usb_descriptors`) and destroyed a second time at the end of the bracket (L.805)